khmer/kfile.py

```diff
@@ -22,7 +22,16 @@ def check_file_status(file_path, force):
 
     if file_path is '-':
         return
-    mode = os.stat(file_path).st_mode
+    try:
+        mode = os.stat(file_path).st_mode
+
+    except OSError as error:
```

Comment:
`W0612: (unused-variable), check_file_status: Unused variable 'error'` -> no need for `as error` here